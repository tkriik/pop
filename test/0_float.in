0.0 1.23 +5.5 -4.2
. . . .
