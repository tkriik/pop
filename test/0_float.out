-4.2000000000000002
5.5
1.23
0
